client/lib/screen-title/index.js

```diff
@@ -13,16 +14,21 @@ import { setTitle as legacySetTitle } from './actions';
 export function subscribeToStore( store ) {
```

Comment:
* I knew about the `let` vs. `const`, but thanks for confirming.
* Didn't know that, assigning it only once should be a better solution.
* Is there a documentation to which selectors are available? I know there is a lot of documentation but I couldn't find it anywhere obvious.
* It's a bit boring to look at, but creating another helper property only seems to complicate things to me.
* Setting the state as a `const` didn't work just after the function declaration, it does work however when setting it in the `subscribe()` definition.